torch_xla/csrc/unwrap_data.cpp

```diff
@@ -0,0 +1,50 @@
+#include <ATen/Functions.h>
+#include <torch/csrc/lazy/backend/backend_data.h>
+#include <torch/csrc/lazy/backend/backend_interface.h>
+#include <torch/csrc/lazy/core/metrics.h>
+#include <torch/csrc/lazy/core/util.h>
+
+#include <memory>
+#include <vector>
+
+#include "absl/types/span.h"
+#include "third_party/xla_client/computation_client.h"
+#include "torch_xla/csrc/xla_data.h"
+
+namespace torch_xla {
+
+xla::ComputationClient::DataPtr UnwrapXlaData(
```

Comment:
Let's also get rid of it in later pr, if xla_client can inherit from pytorch Data class directl.y